content/en/docs/tasks/debug/debug-application/debug-running-pod.md

```diff
@@ -628,8 +628,11 @@ When creating a debugging session on a node, keep in mind that:
 
 * `kubectl debug` automatically generates the name of the new Pod based on
   the name of the Node.
-* The container runs in the host IPC, Network, and PID namespaces.
 * The root filesystem of the Node will be mounted at `/host`.
```

Comment:
I'm wondering if mounting the root filesystem of the Node is okay if the pod is not privileged. I may be wrong though.

Reply:
I agree the behavior is a bit strange; what I called a sort of pseudo-privileged in another comment. But removing the mount would be a default change and would make the command much less useful. If it was changed, one of my projects would break.

Reply:
+1 @kgibm